securedrop_client/logic.py

```diff
@@ -596,25 +611,37 @@ def run_export_preflight_checks(self):
         '''
         Run preflight checks to make sure the Export VM is configured correctly and
         '''
-        logger.debug('Running export preflight checks')
+        logger.debug('Calling export preflight checks from thread {}'.format(threading.current_thread().ident))
 
-        if not self.qubes:
-            return
+        # temporarily commented for testing
+        # if not self.qubes:
+        #    return
 
-        self.export.run_preflight_checks()
+        # Run using a separate thread context (a reference to the thread is
```

Comment:
Concept looks good and it does get us closer to a design where we can use a persistent queue (aka offline queue) so that a user can start more than one export job at a time.

Done with my initial pass of this draft PR.